src/utils/sign_utils.ts

```diff
@@ -30,3 +34,59 @@ export function getFreezeIssuanceTypedData(networkId: number, securityTokenAddre
 export function getDisableControllerTypedData(networkId: number, securityTokenAddress: string) {
   return getTypedData(networkId, securityTokenAddress, 'disabling controller');
 }
+
+export function getHashedTransferData(
+  tmAddress: string,
+  investorsData: InvestorTransferData[],
+  validFrom: Date,
+  validTo: Date,
+  nonce: number,
+) {
+  const investorAddresses = investorsData.map(i => i.investorAddress);
```

Comment:
Same 4 lines of code is being called below, not sure if there is a way to share this between them

Reply:
Solved. Extracted to private function.